src/components/sokperson/SokPerson.tsx

```diff
@@ -40,6 +62,12 @@ export default function SokPerson() {
   } = useSokPerson();
   const [isFormError, setIsFormError] = useState<boolean>(false);
 
+  useEffect(() => {
+    if (isSuccess) {
+      logSokPersonResults(searchResults.length);
+    }
+  }, [isSuccess, searchResults]);
```

Comment:
Kunne man løst det uten en useEffect? Men er kanskje riktig, siden det er en sideeffekt.

Reply:
`mutate` har vel en sånn `onSuccess` greie🤔 Går det an å logge det der? 😁 Se `TildeltVeileder` for eksempel

Reply:
Ja true, det er jo kanskje bedre 👍🏼

Reply:
Ikke noe galt med `useEffect` her tror jeg, men synes å legge det i `onSuccess`-callbacken på `mutate` leser bedre :+1:

Reply:
Det var smart, funka bra 👍🏼